### ISSUE TITLE
Why int rather than short for packed output variables?

### ISSUE THREAD

**Issue by kafitzgerald** [NCAR/wrf_hydro_nwm_public#511]
We currently store packed output variables using scale + offset as `int`.  Ideally these would be stored as `short`.  

Just another thing to look into re: outputs

**Comment by rcabell** [comment]
Agreed, using int vs float saves no (uncompressed ) space as both are 32-bit.
